CMakeLists.txt

```diff
@@ -1,8 +1,23 @@
-project(ObjRecRANSACRun)
-cmake_minimum_required(VERSION 2.8.3)
```

Comment:
Probably doesn't need to be a separate project. This was a year or two ago so I don't remember for certain